spring-app/src/main/java/pl/edu/agh/cqm/controller/MonitoringController.java

```diff
@@ -70,4 +70,19 @@ public AllParametersResponseDTO getAll(
                 monitoringService.getThroughputSamples(startDate, endDate)
         );
     }
+
+    @PutMapping("/update-parameters")
```

Comment:
updating a REST resource should look something like `PUT /api/path/to/resource`, so in this case `PUT /api/parameters`